packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -25,7 +25,7 @@ import { RumEvent } from '../rumEvent.types'
 import { buildEnv } from './buildEnv'
 import { startRum } from './startRum'
 
-export interface RumInitConfiguration extends InitConfiguration {
+export interface RumInitConfiguration extends Omit<InitConfiguration, 'publicApiKey' | 'datacenter'> {
```

Comment:
You should import Omit from utils to not break compatibility with older ts version

Reply:
Yes